java/ql/lib/change-notes/2022-02-21-type-hierarchy.md

```diff
@@ -0,0 +1,6 @@
+---
+category: feature
+---
+* Added `hasDescendant(RefType anc, Type sub)`
+* Added `RefType.getADescendant()`
+* Added `RefType.getAStrictAncestor()`
```

Comment:
```suggestion
* Added `RefType.getAnAncestor()`
* Added `RefType.getAStrictAncestor()`
```

Reply:
`RefType.getAnAncestor()` already exists; this just changes its definition.